src/tests/ilverify/ILTests/FieldTests.il

```diff
@@ -28,6 +28,30 @@
     .field public initonly int32 InstanceInitonlyField
     .field public static initonly int32 StaticInitonlyField
 
+   .field private initonly int32 '<MyProperty>backingField'
+
+    .custom instance void [System.Runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (01 00 00 00)
```

Comment:
I would delete these CompilerGeneratedAttributes. They are just a distraction, not necessary for the test to work.